helm/rbac-operator/values.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "image": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                }
+            }
+        },
+        "oidc": {
+            "type": "object",
+            "properties": {
+                "customer": {
+                    "type": "object",
+                    "properties": {
+                        "write_all_group": {
```

Comment:
Recently we introduced a possibility to define multiple groups of a specific type (customer or giantswarm). For that purpose the operator also supports plural version of the `write_all_group` property: `write_all_groups`, which expects an array of objects. Each object has a single property called `name`. No other properties are supported atm, but we may need to add some in the future.
Singular `write_all_group` are still supported to ensure backward compatibility.
So the new `write_all_groups` properties probably need to be added to the schema as well.

Reply:
Thanks for pointing that out. I have added `write_all_groups` to the `values.yaml` and `values.schema.json`.